### PR TITLE
Create style.css

### PR DESCRIPTION
**Pull request by jason-whitted**
Created a simple stylesheet with a header

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+h1 {
+  background-color: silver;
+  border: 1px solid blue;
+  box-shadow: 0 0 5px red;
+  text-align: center;
+}
```
Comment (etalisoft) [inline assets/css/style.css:6]: Could you add a blank line to the end of this file?

Comment (etalisoft) [inline assets/css/style.css:6]: According to marketing the H1 tags are supposed to be 50px font.

Comment (jason-whitted) [reply assets/css/style.css:6]: Oh snap, good catch!

**Review by etalisoft — APPROVED** [review]

**Comment by etalisoft** [comment]
Could you address the merge conflicts?

**Pull request merged**